src/telcameras_v2/view_definitions.py

```diff
@@ -4361,6 +5514,8 @@ def get_view_strings(view_name):
     sql_materialized = f"""
         CREATE MATERIALIZED VIEW {view_name}_materialized AS
         SELECT * FROM {view_name};
+        
+        CREATE UNIQUE INDEX {view_name}_materialized_sensor_timestamp_rounded_idx ON public.{view_name}_materialized USING btree (sensor, timestamp_rounded);
```

Comment:
Op een bestaande installatie heeft dit alleen effect op nieuwe views, terwijl dit bij een nieuwe installatie ook op alle voorgaan views wordt gedaan. Dit lijkt me wel iets wat nog even wat verder uitgedacht moet worden. Misschien moet dit in een apart commando komen zodat we het in de migratie aan kunnen roepen wanneer nodig.